docs/Type-Providers.md

```diff
@@ -85,4 +85,6 @@ server.get('/route', {
 })
 ```
 
+TypeBox uses the properties `kind` and `modifier` internally. Those are not strictly valid JSON schema and will throw an AJV error. To remove the error it's either necessary to omit the properties by using [`Type.strict()`](https://github.com/sinclairzx81/typebox#strict) or use the AJV options for adding custom keywords.
```

Comment:
`Type.strict()` should be capitalized as `Type.Strict()`

Reply:
Possibly rephrase 

> Those are not strictly valid JSON schema and will throw an AJV error

 as 

> These properties are not strictly valid JSON schema which will cause AJV to throw an invalid schema error

Reply:
Thanks! Comments are incorporated now :)